tests/main/snap-quota-groups/task.yaml

```diff
@@ -129,9 +129,9 @@ execute: |
   # system is working we won't fail the test simply because systemd won the race
   retry --wait 1 -n 10 sh -c 'systemctl show --property=ExecMainStatus snap.go-example-webserver.webserver.service | MATCH "ExecMainStatus=(203|9)"'
 
-  echo "And after removing the quota group, the snap is restarted and active again"
+  echo "And after removing the quota group, services will not restart as they were inactive"
   snap remove-quota too-small
-  retry --wait 1 -n 10 sh -c 'snap services go-example-webserver.webserver | MATCH "go-example-webserver.webserver\s+enabled\s+active"'
+  snap services go-example-webserver.webserver | MATCH "go-example-webserver.webserver\s+enabled\s+inactive"
```

Comment:
I think this is fine, but it should also do `snap start go-example-webserver.webserver` and then check that the service is active again, probably needs to be checked in a loop.

If we really wanted to be robust here, we could count the number of times that the service had tried to start before removing the quota group, and then check the start count again after removing the quota group and make sure they are the same to demonstrate that indeed no attempt was made to start the inactive service here.

Reply:
> I think this is fine, but it should also do `snap start go-example-webserver.webserver` and then check that the service is active again, probably needs to be checked in a loop.

Done.

> If we really wanted to be robust here, we could count the number of times that the service had tried to start before removing the quota group, and then check the start count again after removing the quota group and make sure they are the same to demonstrate that indeed no attempt was made to start the inactive service here.

For this I now added a check on the startup timestamp. Please let me know if you have better suggestions :-)